user/main.cpp

```diff
@@ -30,7 +30,7 @@ std::string GetCRC32(std::filesystem::path filePath) {
 		auto readSize = fin.gcount();
 		crc32.add(&buffer[0], (size_t) readSize);
 	}
-	LOG_DEBUG("CRC32 of \"" + filePath.u8string() + "\" is " + crc32.getHash());
+	//LOG_DEBUG("CRC32 of \"" + filePath.u8string() + "\" is " + crc32.getHash());
```

Comment:
Only thing I have a question about is this- why'd we remove the debug message showing the CRC?

Outside of this everything looks good (I say that but I haven't looked through the il2cpp types file). Will approve my review.